### ISSUE TITLE
Give access to country channels for limited time

### ISSUE THREAD

**Issue by hasanlatch007** [Yes-Theory-Fam/yesbot-ts#499]
I've seen a lot of messages whenever a user is asking for access to certain country channels commands, and once #495 is merged `#travel-together` will be locked to users
If possible could we mix the timer event and the DB to give them limited access on a specified timer for the requested country?
Could be like 5 minutes and then he will have to wait 48 hours before he could use the command again

Another implementation would be for us to use the `!menu` command, where the user can send one message to the channel and the command can only be used every 24 hours. 
The message is sent in the channel, add reaction if the user wants to initiate a conversation with the user asking his question, create dm group with the two.
Or simply scratch the last idea and just make the user manually message him :p

**Comment by geisterfurz007** [comment]
This is planned indeed for the future(-ish) but not for free ;)

I will mark this as blocked for now since this should not be implemented before people can pay for their flights to other country channels :)

**Comment by geisterfurz007** [comment]
Closing this in favour of #566 since that is in the pulk with the other currency features.
